src/main/java/util/DateTimeUtil.java

```diff
@@ -0,0 +1,47 @@
+package util;
+
+import java.time.DateTimeException;
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+
+import domain.Holiday;
+import dto.Formatter;
+
+public class DateTimeUtil {
+
+    public static boolean isOffDay(LocalDate date) {
+        boolean isWeekend = date.getDayOfWeek().equals(DayOfWeek.SATURDAY) ||
+            date.getDayOfWeek().equals(DayOfWeek.SUNDAY);
+        return isWeekend || isHoliday(date);
+    }
+
+    private static boolean isHoliday(LocalDate date) {
+        return !Holiday.from(date).equals(Holiday.NONE);
```

Comment:
enum 은 보통 equals 보다는 == 비교를 사용합니다. 🙂 

```suggestion
        return Holiday.from(date) != Holiday.NONE;
```

Reply:
enum을 ==으로 비교해볼 생각은 못했네요!
enum을 비교할 때 equals보다 == 비교를 선호하는 엘리만의 이유가 무엇인지 궁금해요!

Reply:
저만의 이유는 아니고, `==` 비교가 더 안전하기 때문입니다.

아래 블로그를 참고해보세요!
* https://chan9.tistory.com/174

Reply:
NPE 발생에 차이가 있었군요!
도움이 되었습니다. 감사합니다 😄